packages/fluentui/react-northstar/src/components/Tree/TreeItem.tsx

```diff
@@ -265,6 +266,18 @@ export const TreeItem: ComponentWithAs<'div', TreeItemProps> & FluentComponentSt
     _.invoke(props, 'onClick', e, props);
   };
 
+  const handleKeyDown = (e: React.KeyboardEvent) => {
+    e.preventDefault();
+    e.stopPropagation();
+    if (e.key && e.key.length === 1 && e.key.match(/\S/) && e.key !== '*') {
```

Comment:
would be worth testing that these keys do not trigger a search

Reply:
now that we support all keys in the handler :)

Reply:
Yeah, I suggest to add a E2E with Cyrillic characters to ensure that it works 👍